frameworks/g-cloud-9/questions/services/multiqAPIHosting.yml

```diff
@@ -1,18 +1,17 @@
+id: multiqAPI
```

Comment:
Not sure we need to specify an ID for the `multiq` questions, as those keys shouldn't be saved in our data at all.  But I don't think it will hurt, either?